ubyssey/static/src/styles/modules/_archive.scss

```diff
@@ -0,0 +1,27 @@
+.bars {
```

Comment:
All the styles in this file should ideally be under a single parent (i.e. `.archive`) to prevent conflicts with styles on other parts of the site.